lib/project_types/script/script_project.rb

```diff
@@ -6,6 +6,7 @@ class ScriptProject < ShopifyCli::Project
     SOURCE_DIR = "src"
 
     attr_reader :extension_point_type, :script_name, :language
+    alias_method :folder, :script_name
```

Comment:
What does this do?

Reply:
Makes `folder` an alias method of `script_name`. I was hoping to make an explicit connection between the folder name and script name, but I actually just ended up deleting this